webcompat/static/js/lib/wizard/steps/tested-browsers.js

```diff
@@ -29,13 +31,15 @@ const handleNext = (event) => {
 
 const handleNoOther = (event) => {
   event.preventDefault();
-  resetRadio(radio);
+  resetBrowserSelection(radio);
   makeAStep("warningBrowser");
+  notTestedOtherRadio.prop("checked", true);
```

Comment:
I had to re-teach myself:

https://github.com/webcompat/webcompat.com/blob/ad693004dcc017fc56b7a2e85f287948db0075a2/webcompat/form.py#L499-L500 is what this is setting, which ends up here in the template: https://github.com/webcompat/webcompat.com/blob/ad693004dcc017fc56b7a2e85f287948db0075a2/webcompat/form.py#L513